### PR TITLE
Create awu-ec-nomination-notice.en.md

### PR DESCRIPTION
**Pull request by shiibainu**
Adding a new page for the nomination notice.

### CODE REVIEW

File: frontend/content/awu-ec-nomination-notice.en.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+**Petition Submission Deadline: *4 PM PST, August 4th, 2022.*** Petitions must be submitted in a single email to elections@alphabetworkersunion.org. Petitions not received by the deadline and/or partial petitions will not be considered.
+
+Petition Requirements: For this election, the CWA Local 1400 petition process as described in the bylaws has been modified to ensure members have a reasonable opportunity to nominate candidates, given the current COVID-19 health concerns and limited number of members in workplaces.
+
+Petition signatures must be those of members in good standing (see below for definition).
+
+For this election, a petition “signature” is either:
+
+<ol type="a">
```
Comment (jonahbron) [inline frontend/content/awu-ec-nomination-notice.en.md:51]: Unless you verified this works locally, I'm pretty sure HTML directly in a markdown file will not work like this.  You can achieve this by using the literal "a. a hand-written ..." syntax.

**Review by jonahbron — CHANGES_REQUESTED** [review]

**Review by jonahbron — APPROVED** [review]

File: frontend/content/awu-ec-nomination-notice.en.md (hunk 1 above)
Comment (jonahbron) [inline frontend/content/awu-ec-nomination-notice.en.md:51]: Unless you verified this works locally, I'm pretty sure HTML directly in a markdown file will not work like this. You can achieve this by using the literal "a. a hand-written ..." syntax.

**Review by jonahbron — CHANGES_REQUESTED** [review]

**Comment by Dilnu** [comment]
Looks right to me.

**Comment by shiibainu** [comment]
Thanks Samuel! I think this is fine to merge if all else looks well.

File: frontend/content/_index.en.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ date: 2020-12-14T22:17:11-08:00
 {{< home-slider >}}
 
 {{< public-notice >}}
-[Nomination notice for AWU Executive Council (2022-07-04)](/docs/notices/CWA_1400-AWU_NOMINATION_NOTICE_2022.pdf)
+[Nomination notice for AWU Executive Council (2022-07-04)](awu-ec-nomination-notice.en.md)
```
Comment (jonahbron) [inline frontend/content/_index.en.md:10]: I don't think this link will work out of the box.  You need the ref shortcode in markdown.

https://gohugo.io/content-management/cross-references/

**Review by jonahbron — CHANGES_REQUESTED** [review]

File: frontend/content/_index.en.md (hunk 2 above)
Comment (shiibainu) [reply frontend/content/_index.en.md:10]: to make sure I understand correctly, does that mean I should replace the link with
({{< ref "awu-ec-nomination-notice.en.md" >}})
?

Comment (jonahbron) [reply frontend/content/_index.en.md:10]: Yes, I think that should work

**Review by jonahbron — APPROVED** [review]

**Pull request merged**